src/components/InstancesList/InstancesList.test.js

```diff
@@ -141,27 +143,18 @@ describe('InstancesList', () => {
       expect(document.querySelectorAll('#pane-results-content .mclRowContainer > [role=row]').length).toEqual(3);
     });
 
-    it('should have selected browse call number option', () => {
-      fireEvent.change(screen.getByRole('combobox'), {
-        target: { value: 'callNumbers' }
-      });
-
+    it('should have selected browse call number option', async () => {
+      await userEvent.selectOptions(screen.getByLabelText('Search field index'), 'callNumbers');
```

Comment:
looks like switching to `userEvent` might have caused things to progress faster than they previously were.
I can see these failures locally, and wrapping the `expect` statement in a `waitFor` call allowed them enough time to pass.
```
waitFor(() => { 
  expect((screen.getByRole('option', { name: 'Browse call numbers' })).selected).toBeTruthy(); 
});
```

Reply:
I'm uncertain, but the timing difference may also have come from the lifecycle refactor in searchAndSort - though I'm not sure on the specifics - the select changes the state which feeds it back is value... this is re-confirmed when the query parameters come back to the component, so... 🤷

Reply:
That's interesting info, @JohnC-80, thanks. I've also found that I can repeat the failures locally using the `yarn.lock` from CI. I'm trying to figure out if I can narrow the problem to a specific version change now.

Reply:
`waitFor` FTW. Thanks, @JohnC-80! Still, I want to figure out why this broke. Building with an old `yarn.lock` file broke tests, but then pinning to a newer stripes-smart-components left it still broken, suggesting something else is the culprit. 

Regardless, @mkuklis, this is good to merge now!